src/test/java/edu/cornell/kfs/rass/batch/xml/fixture/RassXmlAwardEntryFixture.java

```diff
@@ -188,6 +208,13 @@ public RassXmlAwardEntry toRassXmlAwardEntry() {
         award.setOrganizationCode(organizationCode);
         award.setCostShareRequired(costShareRequired);
         award.setFinalReportDueDate(buildDateFromDateTime(finalReportDueDate));
+        award.setEverify(everify);
+        award.setFinalFinancialReportRequired(finalFinancialReportRequired);
+        award.setBudgetStartDate(buildDateFromDateTime(budgetStartDate));
+        award.setBudgetStopDate(buildDateFromDateTime(budgetStopDate));
+        award.setPricingType(pricingType);
+        award.setPrimeAgreementNumber(primeAgreementNumber);
```

Comment:
The code above doesn't appear to be setting the budget total on the XML Award DTO.